src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/LanguageServer/Handler/Restore/RestoreHandler.cs

```diff
@@ -49,13 +49,18 @@ private async Task RestoreAsync(ImmutableArray<string> pathsToRestore, BufferedP
     {
         foreach (var path in pathsToRestore)
         {
-            var arguments = $"restore \"{path}\"";
+            var arguments = new string[] { "restore", path };
```

Comment:
Didn't want to inline with collection expressions here?

Reply:
There appear to be a bunch of formatting issues with collection expressions.  Every time I try to use them I get formatting warnings (even with 'correct' formatting).  I'll file a bug on that one sec.